src/tokens/solana.tokenlist.json

```diff
@@ -15689,62 +15689,76 @@
+      "name": "Graviton",
+      "decimals": 8,
+      "logoURI": "https://v1.graviton.one/img/brand/download/graviton-logo/primary-logo-png.png",
+      "tags": [],
+      "extensions": {
+        "website": "https://graviton.one",
+        "twitter": "https://twitter.com/OneGraviton",
+        "blog": "https://medium.com/graviton-one",
+        "telegram": "https://t.me/graviton_community",
+        "coingeckoId": "graviton"
+      }
+    },
+    {
       "chainId": 101,
       "address": "4qv2AGjhzxiNz5iTUnTdRVYjjacmaEcVGgFcr1R6sRFe",
       "symbol": "NXMC",
       "name": "NextMindCoin",
       "decimals": 7,
       "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/4qv2AGjhzxiNz5iTUnTdRVYjjacmaEcVGgFcr1R6sRFe/logo.png",
       "tags": [
-	     "utility-token"
-	  ],
+        "utility-token"
+      ],
       "extensions": {
         "website": "https://www.nextmindcoin.com",
         "twitter": "https://twitter.com/nextmindcoin",
         "discord": "https://discord.gg/YXDNHMadC4",
-		"telegram": "https://t.me/NextMindCoin_NXMC",
-		"github": "https://github.com/CEO-NextMindCoin"
-      }
-      },
-    {
-       "chainId": 101,
-       "address": "E6UU5M1z4CvSAAF99d9wRoXsasWMEXsvHrz3JQRXtm2X",
-       "symbol": "DGLN",
-       "name": "Dogelana",
-       "decimals": 9,
-       "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/E6UU5M1z4CvSAAF99d9wRoXsasWMEXsvHrz3JQRXtm2X/logo.png",
-       "tags": ["dogecoin", "dogelana", "doge", "solana", "meme", "dgln"],
-       "extensions": {
-         "website": "http://www.dogelana.com",
-         "twitter": "https://twitter.com/dogelana",
-         "blog": "http://dogelana.com/blog",
-         "telegram": "https://t.me/dogelana",
-         "facebook": "https://facebook.com/dogelana",
-         "coingeckoId": "dogelana",
-         "email": "info@dogelana.com",
-         "coinmarketcap": "https://coinmarketcap.com/currencies/dogelana/"
-       }
+        "telegram": "https://t.me/NextMindCoin_NXMC",
+        "github": "https://github.com/CEO-NextMindCoin"
+      }
+    },
+    {
+      "chainId": 101,
+      "address": "E6UU5M1z4CvSAAF99d9wRoXsasWMEXsvHrz3JQRXtm2X",
+      "symbol": "DGLN",
+      "name": "Dogelana",
+      "decimals": 9,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/E6UU5M1z4CvSAAF99d9wRoXsasWMEXsvHrz3JQRXtm2X/logo.png",
+      "tags": [ "dogecoin", "dogelana", "doge", "solana", "meme", "dgln" ],
+      "extensions": {
+        "website": "http://www.dogelana.com",
+        "twitter": "https://twitter.com/dogelana",
+        "blog": "http://dogelana.com/blog",
+        "telegram": "https://t.me/dogelana",
+        "facebook": "https://facebook.com/dogelana",
+        "coingeckoId": "dogelana",
+        "email": "info@dogelana.com",
+        "coinmarketcap": "https://coinmarketcap.com/currencies/dogelana/"
+      }
+    },
+    {
+      "chainId": 101,
```

Comment:
I want to add just this KPLR  (KEPLER PAY) Token, I am not adding  or deleting anything else. 

,
    {
      "chainId": 101,
      "address": "DTQbtBQUWC6cbw8TD83ma8SWy1oHux8stZsMrXWoyaLp",
      "symbol": "KPLR",
      "name": "KEPLER PAY",
      "decimals": 11,
      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/DTQbtBQUWC6cbw8TD83ma8SWy1oHux8stZsMrXWoyaLp/logo.png",
      "tags": [
        "coin"
      ],
      "extensions": {
        "website": "https://wtmakers.com"
      }
    }